commands/command_prune.go

```diff
@@ -0,0 +1,460 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
+
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
+)
+
+var (
+	pruneCmd = &cobra.Command{
+		Use:   "prune",
+		Short: "Deletes old LFS files from the local store",
+		Run:   pruneCommand,
+	}
+	pruneDryRunArg      bool
+	pruneVerboseArg     bool
+	pruneVerifyArg      bool
+	pruneDoNotVerifyArg bool
+)
+
+func pruneCommand(cmd *cobra.Command, args []string) {
+
+	// Guts of this must be re-usable from fetch --prune so just parse & dispatch
+	if pruneVerifyArg && pruneDoNotVerifyArg {
```

Comment:
It feels weird having both --verify-remote and --no-verify remote. I assume it's part of the prune spec? Maybe we could have setting one to true set the other to false so that we don't always have to check two things?